2013/data_file_specification.json

```diff
@@ -260,6 +390,9 @@
                "end": "54",
                "length": "5",
                "dataType": "N",
+               "format": "currency,thousands",
```

Comment:
My only thought here would be to perhaps use `;` instead of `,` as the separator, so that it is easier to distinguish by a human when reading the JSON.